src/services/safe-config/safe-config.module.ts

```diff
@@ -2,9 +2,14 @@ import { HttpModule } from '@nestjs/axios';
 import { Module } from '@nestjs/common';
 import { SafeConfigService } from './safe-config.service';
 
+const BASE_URL_PROVIDER = {
+  provide: 'BASE_URL',
```

Comment:
This is module-scoped, right? Otherwise we could use a prefix, like CONFIG_SERVICE_BASE_URL :)

Reply:
Yeah good point. I actually would prefer to have it as just `BASE_URL_PROVIDER` but scoped only to this module. It should not be possible for this to be injected elsewhere directly.

Reply:
Done in [2364b76](https://github.com/5afe/safe-client-gateway-nest/pull/5/commits/2364b76f6a01f8ed1f74f44768a677a166dcfe0d)